.circleci/config.yml

```diff
@@ -364,11 +367,76 @@ commands:
           det-version: <<parameters.det-version>>
           agent-instance-type: <<parameters.agent-instance-type>>
           max-dynamic-agents: <<parameters.max-dynamic-agents>>
-      - set-master-address:
+      - set-master-address-aws:
           cluster-id: ${CLUSTER_ID}
       - wait-for-master:
           host: ${MASTER_HOST}
 
+  terminate-gke-cluster:
+    parameters:
+      cluster-id:
+        type: string
+    steps:
+      # Use a run instead of `gke/delete-cluster` because circle CI orbs do not support `when`.
+      - run:
+          name: Terminate GKE Cluster
+          when: always
+          command: |
+            gcloud container clusters delete <<parameters.cluster-id>> --quiet --region=us-west1
```

Comment:
Here you should also add a `region` parameter.

Reply:
done